java/src/client/server/ClientCommunicator.java

```diff
@@ -50,11 +55,19 @@ public String sendHTTPRequest(String URLSuffix, String requestBody, String reque
             URL url = new URL(URLPrefix + URLSuffix);
             connection = (HttpURLConnection) url.openConnection();
             connection.setRequestMethod(requestMethod);
+            /*if (msCookieManager.getCookieStore().getCookies().size() > 0) {
```

Comment:
Don't have commented code unless it's guaranteed to be short and temporary. Use git to retrieve history if code is not going to be used for now. Please remove this.